src/main/resources/nl/tudelft/ti2206/group9/level/tp1_coinlaneleft.txt

```diff
@@ -0,0 +1,100 @@
+...
```

Comment:
These files are outdated. These old coinlanes probably remained from a merge commit. Please remove the old coinlanes.

<organization